conformance_pack/ec2.sp

```diff
@@ -234,6 +259,17 @@ control "ec2_instance_no_launch_wizard_security_group" {
   })
 }
 
+control "ec2_instance_virtualization_type_no_paravirtual" {
+  title       = "Paravirtual EC2 instance types should not be used"
+  description = "This control checks whether the virtualization type of an EC2 instance is paravirtual. The control fails if the virtualizationType of the EC2 instance is set to paravirtual."
+  query       = query.ec2_instance_virtualization_type_no_paravirtual
+
+  tags = merge(local.conformance_pack_ec2_common_tags, {
+    nist_csf     = "true"
+    other_checks = "true"
```

Comment:
```suggestion
    nist_csf     = "true"
```

Reply:
This happened while resolving merge conflict.